lonboard/_map.py

```diff
@@ -167,6 +167,44 @@ def __init__(
       [`lonboard.basemap.CartoBasemap.PositronNoLabels`][lonboard.basemap.CartoBasemap.PositronNoLabels]
     """
 
+    custom_attribution = traitlets.Union(
+        [
+            traitlets.Unicode(allow_none=True),
+            traitlets.List(traitlets.Unicode(allow_none=False)),
+        ]
+    ).tag(sync=True)
```

Comment:
Does the attribution need to support hyperlinking too? I.e. do we need to define not just text but also the target URL?

Reply:
@kylebarron the MapLibre attribution control renders html links if the attribution text contains an <a> element. They are not appearing now because the Map container has `z-index: -1`, making the DeckGL canvas to be placed in front of the map controllers, blocking mouse events on it. In the recording below I disabled the `z-index`, but that caused the data layer to disappear. The z-index is not defined in Lonboard code, I would suggest we address this issue in separate because resolving the z-index conflict might require a more complex solution that ensures both the DeckGL canvas and MapLibre controllers are correctly rendered and interactive.

![hyperlink](https://github.com/developmentseed/lonboard/assets/329694/c61d77cf-c6a1-47c5-8bc1-6b4f8d7e343d)

Reply:
Right. I think this is again tied to #437. The "proper" way as of the latest deck.gl versions is to use MapboxOverlay, which has the pros and cons described in #437. I think when you use MapboxOverlay then the maplibre attribution control is clickable.

Reply:
Apologies, I haven't looked into #437 in depth. I believe we can implement a custom attribution control, which would untie us from MapLibre. We would need a way to extract attribution from map styles JSON, but I think it is doable. Should we keep discussing this on #561?